lib/Data/Validate/Sanctions.pm

```diff
@@ -43,8 +45,18 @@ sub is_sanctioned {        ## no critic (RequireArgUnpacking)
     return (get_sanctioned_info(@_))->{matched};
 }
 
-sub get_sanctioned_info {    ## no critic (RequireArgUnpacking)
+sub _possible_match {
```

Comment:
I know its not mentioned in the Perl Style guide but Toms pref is for private functions to be at the bottom of the file.  IMO they should also have the  standard subroutine documentation.

Reply:
https://binary-group.slack.com/archives/C0D442KHV/p1536887868000100